SITCOMTN-076.tex

```diff
@@ -49,6 +53,325 @@
 % ADD CONTENT HERE
 % You can also use the \input command to include several content files.
 
+\begin{warning}[Drafting in progress!]
+  This policy document is not yet released.
+\end{warning}
+
+\section{Motivation and Scope}
+
+The final stages of the Rubin Observatory Construction Project involving on-sky commissioning and science validation of the integrated system will be an exciting and dynamic time.
+During this period, new capabilities will be continuously added and tested.
+Our understanding of the technical and scientific performance of the Observatory will be rapidly advancing.
+In parallel, the operations team will be preparing to start the LSST 10-year survey and to release on-sky data products to the science community as part of the Early Science Program \citedsp{RTN-011}.
+
+For multiple reasons, Rubin Observatory needs clear and well-publicized policies on information sharing as a function of time during the on-sky commissioning phase.
+
+\begin{enumerate}
+
+  \item The Rubin Observatory has a set of standards for data security.
+  The relevant aspects are as follows:
+  %The US funding agencies, National Science Foundation (NSF) and Department of Energy (DOE), have provided a set of standards for data security (DMTN-199).
+
+  \begin{itemize}
+
+    \item Delayed public release of focal plane scientific data:
+    During commissioning, engineering and imaging data will be embargoed for all non-Project team members for a period of at least 30 days following the observation.
+    After this 30 day embargo, only with explicit approval may proprietary information, including data products from commissioning, be shared outside the Project team.
+    Data aside from focal plane scientific data may be made available following the Project plan which includes astronomical metadata (within 24 hours), alert postage stamp images (within 60 sec), and weather and sky monitoring data.
```

Comment:
What does "astronomical metadata" mean?  ra/dec/seeing?

Reply:
Details appear to be included in https://dmtn-263.lsst.io/ describing ObsLocTap. Do we want to get into that much detail in this document?